.changelog/2650.internal.1.md

```diff
@@ -0,0 +1,9 @@
+replace Roothash, Scheduler and Registry fields with Consensus accessors
+
+ConsensusAPI.Backend contains accessors for RootHash, Scheduler, and Registry.
+Use those instead of direct references. The following structs were affected:
+- Node in cmd/node/node.go,
```

Comment:
Perhaps even better than referencing file paths would be to reference Go packages here.

Reply:
Good idea. I was just thinking about this...